scripts/coderabbit_retry/coderabbit_retry.py

```
@@ -0,0 +1,384 @@
+    """Return whether a PR title indicates work in progress.
+
+    Args:
+        title: Pull request title.
+
+    Returns:
+        ``True`` if the title contains ``wip`` as a word or ``[wip]``.
+    """
+    return bool(re.search(r"\bwip\b|\[wip\]", title, re.IGNORECASE))
+
+
+def main() -> int:
```

Ruff (0.16.1)

[warning] Too many branches (17 > 12)

(PLR0912)